includes/class-wc-pakettikauppa.php

```diff
@@ -100,7 +100,7 @@ public function pickup_point_field_html() {
 
       $chosen_shipping_id = explode( ':', WC()->session->get( 'chosen_shipping_methods' )[0] )[1];
 
-      $shipping_method_id = $shipping_methods[ $chosen_shipping_id ]->instance_settings['shipping_method'];
+      $shipping_method_id = $shipping_methods[ strval($chosen_shipping_id) ]->instance_settings['shipping_method'];
```

Comment:
Are you sure this is the correct fix to https://github.com/Seravo/woo-pakettikauppa/issues/48 ? Would not the correct fix be to access the array elements by name?

Reply:
It is accessing the array element by it's name - that is why strval -function call for the chosen_shipping_id (which is int).

Reply:
All tabs and spaces are created using phpcs.xml file and phpcbf command.